public/app/features/explore/LiveLogs.tsx

```diff
@@ -157,7 +157,7 @@ class LiveLogs extends PureComponent<Props, State> {
     const { logsRow, logsRowLocalTime, logsRowMessage } = getLogRowStyles(theme);
 
     return (
-      <>
+      <div>
```

Comment:
This is needed otherwise the transition classes are applied only to first element here.